python/lsst/validate/drp/validate.py

```diff
@@ -88,17 +92,23 @@ def loadAndMatchData(repo, visitDataIds,
     srcVis = SourceCatalog(newSchema)
 
     for vId in visitDataIds:
-        calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
-        calib.setThrowOnNegativeFlux(False)
+        try:
+            calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
+        except TypeError as te:
+            print(te)
+            continue
```

Comment:
Good question.  Now documented in code comment below.  
```
            # DECam images that haven't been properly reformatted
            # can trigger a TypeError because of a residual FITS header
            # LTV2 which is a float instead of the expected integer.
            # This generates an error of the form:
            #
            # lsst::pex::exceptions::TypeError: 'LTV2 has mismatched type'
            #
            # See, e.g., DM-2957 for details.
```

Also added catching FitsError exception when reading FITS files.